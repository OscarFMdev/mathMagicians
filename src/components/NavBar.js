import { NavLink } from 'react-router-dom';

const NavBar = () => (
  <header className="top-section">
    <h1>Math Magicians</h1>
    <nav>
      <NavLink
        to="/"
        className="link"
      >
        Home
      </NavLink>
      <NavLink className="link" to="/calculator">Calculator</NavLink>
      <NavLink className="link" to="/quote">Quote</NavLink>
    </nav>
  </header>
);

export default NavBar;
