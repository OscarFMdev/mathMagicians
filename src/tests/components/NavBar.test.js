import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../../components/NavBar';

describe('NavBar', () => {
  test('should render', () => {
    const { container } = render(<Router><NavBar /></Router>);
    expect(container).toMatchSnapshot();
  });
});
