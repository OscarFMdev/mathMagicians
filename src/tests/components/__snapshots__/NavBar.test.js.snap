// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render 1`] = `
<div>
  <header
    class="top-section"
  >
    <h1>
      Math Magicians
    </h1>
    <nav>
      <a
        aria-current="page"
        class="link active"
        href="/"
      >
        Home
      </a>
      <a
        class="link"
        href="/calculator"
      >
        Calculator
      </a>
      <a
        class="link"
        href="/quote"
      >
        Quote
      </a>
    </nav>
  </header>
</div>
`;
